src/xdg-desktop-portal.c

```diff
@@ -373,6 +374,11 @@ on_bus_acquired (GDBusConnection *connection,
   if (implementation != NULL)
     export_portal_implementation (connection,
                                   account_create (connection, implementation->dbus_name));
+
+  implementation = find_portal_implementation ("org.freedesktop.impl.portal.Email");
+  if (implementation != NULL)
+    export_portal_implementation (connection,
+                                  email_create (connection, implementation->dbus_name));
```

Comment:
Not for now, but I think it would probably make sense to refactor a bit the code in here, as only the D-Bus interface name and the create function are different in all these cases.